/**
 * Created with IntelliJ IDEA.
 * User: glukki
 * Date: 09.05.12
 * Time: 16:30
 * To change this template use File | Settings | File Templates.
 */

import "Flowerbed.proto";
option java_package = "com.kloomba.app.protobuf";
option java_outer_classname = "BookmarkAddProto";

//request: /bookmark/add?id=1
message BookmarkAdd {
    optional int32 timestamp = 1;
    repeated Flowerbed flowerbed = 2; //updated bookmark list
}